gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/HivePurgerWriterBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.compliance;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+
+import gobblin.writer.DataWriter;
+import gobblin.writer.DataWriterBuilder;
+
+
+/**
+ * Initializes {@link HivePurgerWriter} with {@link FileSystem} and {@link HivePurgerQueryExecutor}
+ *
+ * @author adsharma
+ */
+public class HivePurgerWriterBuilder extends DataWriterBuilder<ComplianceRecordSchema, ComplianceRecord> {
+
+  @Override
+  public DataWriter<ComplianceRecord> build()
+      throws IOException {
+    try {
+      return new HivePurgerWriter(FileSystem.newInstance(new Configuration()), new HivePurgerQueryExecutor());
```

Comment:
This assumes that the staging directory is always the default FS. It is better to use WriterUtils.getWriterFS(destination.getProperties())